app/cafe.py

```diff
@@ -0,0 +1,21 @@
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("You not vaccinated!")
```

Comment:
The error message 'You not vaccinated!' is not grammatically correct and could be more descriptive. Consider changing it to something like 'You are not vaccinated!' to better convey the issue.